src/Configurator/DockerComposeConfigurator.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Configurator;
+
+use Symfony\Flex\Lock;
+use Symfony\Flex\Recipe;
+
+/**
+ * Adds services and volumes to docker-compose.yaml file.
```

Comment:
I know I've made this point before, but upstream documentation refers to `docker-compose.yml` everywhere. Sure, `.yaml` works as well, but `docker-compose.yml` is what users would be looking for when using upstream docs. I don't see the need to create additional friction.